### DISCUSSION TITLE
Is the `Special Thanks & A Free Gift from Zed` e-mail real?

### DISCUSSION THREAD

**Discussion by mikaelsouza** [zed-industries/zed#7459] [Feature Requests]
Hey everyone, today I received an e-mail reading:

```
Hey there!

Big thanks for being a superstar supporter of Zed. As a token of our gratitude, we're sending you some exclusive Zed merch for free!

Just fill out this quick form: [removed link] and we'll get your gear on its way.

​
Cheers,
The Zed Team
```

It was sent from `hi@zed.dev`, but I feel like I am about to be scammed. It feels like there's important info missing in the e-mail, like if the shipping is worldwide, for example.

Can some of you share some light about this?

**Comment by Bloeckchengrafik** [comment]
(from the discord)
![image](https://github.com/zed-industries/zed/assets/37768199/34691d0f-104a-46a3-82bc-ebfb9dd923de)

https://discord.com/channels/869392257814519848/873293828805771284/1204562222626635876

**Reply by JosephTLyons** [reply]
Yeah, we started up a [merch store](https://zedindustries.creator-spring.com) and at the moment. We don't have a good way to give out free gear at the moment, so we just made a quick Google Form to get some things out the door. Ideally, we'd have a code or something you just punch in at the website that puts it on our company card.
